index.html

```diff
@@ -8,27 +8,166 @@
+            </div>
+        </div>
+        <div class="Projects">
+            <img class="Projectsimg" src="./Images/proj3.png" alt="Projects Image">
+            <div class="Projectswrp">
+                <p class="projtxt">My Sixth Project</p>
+                <div class="projlang">
+                    <ul>
+                        <li><button type="button" class="smallbtn">HTML/CSS</button></li>
+                        <li><button type="button" class="smallbtn">Ruby On Rails</button></li>
+                        <li><button type="button" class="smallbtn">Javascript</button></li>
+                    </ul>
+                </div>
+                <button type="button" class="projbtn">Check out this Projects<img src="./Images/arrow.png" alt=""></button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="headtitle">
+            <h1 class="mttitle" >About</h1>
+        </div>
+        <div class="maintxt">
+            <p class="mtext" >It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).</p>
+        </div>
+        <button type="button" class="colbtn">Check out my CV <img src="./Images/ic_download.png" alt=""></button>
+        <div class="lang">
+            <div class="sectitle">
+                <h2>Languages</h2>
+            </div>
+            <div class="litxt">
+            <ul>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+            </div>
+        </div>
+        <div class="frames">
+            <div class="sectitle">
+                <h2>Frameworks</h2>
+            </div>
+            <div class="litxt">
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+            </div>
+        </div>
+        <div class="skills">
+            <div class="sectitle">
+                <h2>Skills</h2>
+            </div>
+            <div class="litxt">
+            <ul>
+                <li>Codekit</li>
+                <li>Codepen</li>
+                <li>GitHub</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+            </ul>
+            </div>
```

Comment:
Good job for using ul/li for listing the specificity like Codekit, Codepen ...

For the same reason, you have to use ul/li for listing these 3 elements :  Languages ,Frameworks ,Skills

Reply:
Hello, Languages, Frameworks, and Skills are the titles themselves and each in separate div. Making it li will effect the design, pleaes